docs/sources/operators-guide/configure/configure-otel-collector.md

```diff
@@ -0,0 +1,106 @@
+---
+aliases:
+  - /docs/mimir/latest/operators-guide/configuring/configuring-otel-collector/
+description: Learn how to write data from OpenTelemetry Collector into Mimir
```

Comment:
Should we be more specific and call it "metrics"?

```suggestion
description: Learn how to write metrics from OpenTelemetry Collector into Mimir
```

_If you think metrics is more appropriate, then please do the change below too.